src/components/forms/JurisdictionSelect/index.tsx

```diff
@@ -46,14 +46,39 @@ export interface JurisdictionSelectProps<T = SelectOption> extends AsyncSelectPr
 export const promiseOptions = (
   service: OpenSRPService,
   paramsToUse: Dictionary,
-  hierarchy: SelectOption[]
+  hierarchy: SelectOption[],
+  jurisdictionStatus: boolean,
```

Comment:
What does this relate to?

Reply:
Builds the hierarchy label on the text field